src/test/unit/function-info.test.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { stripFunctionArguments } from '../../content/function-info';
+
+describe('get-function-name', function () {
+  it('should strip the function arguments', function () {
+    expect(stripFunctionArguments('ns::fn()')).toEqual('ns::fn');
+    expect(stripFunctionArguments('ns::fn(bool (*)(JS::Handle<JSObject*>))')).toEqual('ns::fn');
+    expect(stripFunctionArguments('Bar* ns::fn<(Foo)0>(void const*, unsigned int)')).toEqual('Bar* ns::fn<(Foo)0>');
+    expect(stripFunctionArguments('ns::fn(bool, bool) [clone .part.123]')).toEqual('ns::fn');
+    expect(stripFunctionArguments('ns::fn() const')).toEqual('ns::fn');
+    expect(stripFunctionArguments('ns::fn() const [clone .part.123]')).toEqual('ns::fn');
+  });
+  it('should do nothing if not ending with parentheses', function () {
+    expect(stripFunctionArguments('ns::(anonymous namespace)::fn')).toEqual('ns::(anonymous namespace)::fn');
```

Comment:
I don't know. But probably not then. I think I removed the arguments just for testings sake. I'll remove this one.